test/common.py

```diff
@@ -21,31 +21,31 @@
 SEED = 0
 SEED_SET = 0
 ACCEPT = False
+UNITTEST_ARGS = None
 
 
-# TODO rename me
-def parse_set_seed_once():
+def parse_args_set_seed_if_needed():
```

Comment:
Can't we just always run this in `common.py` instead of adding it in every file separately?

Reply:
Good point!